### DISCUSSION TITLE
New Source: Easybill

### DISCUSSION THREAD

**Discussion by ghost** [airbytehq/airbyte#36081] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Source: Easybill
https://www.easybill.de/api/
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
Latest version is fine

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
We need to pull finance data to do some further analysis
* How often do you want to run syncs?
Once a day should be fine
* If this is an API source connector, which entities/endpoints do you need supported?
All of them
* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name?
There is a free trial version

## Describe the alternative you are considering or using
Probably writing some python to pull the data

## Are you willing to submit a PR?
Yes I am

**Comment by marcosmarxm** [comment]
Thanks for requesting this @matfiorlh hope to see your contribution, ask anything here if you have problems submitting the PR.

**Comment by ghost** [comment]
@marcosmarxm I could use some guidance in how to set up things.

**Comment by marcosmarxm** [comment]
Here: https://docs.airbyte.com/connector-development/

[a comment by an automated account is not shown]
